src/OrganisationRegistry.SqlServer/Infrastructure/MemoryCaches.cs

```diff
@@ -129,6 +135,18 @@ public MemoryCaches(IContextFactory contextFactory, ILogger<MemoryCaches>? logge
             }
         }
 
+        internal IList<Guid> GetSparseCache<T>(MemoryCacheType cacheType)
```

Comment:
wat wil dit zeggen ?

Reply:
Ivm gewone cache die true/false voor alles opslaat, slaan we enkel die op die relevant zijn

Reply:
ah ok